FWCore/ParameterSet/python/Types.py

```diff
@@ -40,11 +40,27 @@ def __init__(self,type):
         super(_ProxyParameter,self).__init__()
         self.__dict__["_ProxyParameter__type"] = type
         self.__dict__["_ProxyParameter__value"] = None
+        if hasattr(self.__type,"_default") and self.__type._default is not None:
+            self.setValue(self.__type._default)
     def setValue(self, value):
         v = self.__type(value)
         if not _ParameterTypeBase.isTracked(self):
             v = untracked(v)
         self.__dict__["_ProxyParameter__value"] = v
+    def _returnTypeWithValue(self, valueWithType):
```

Comment:
Does the "type with value" and "value with type" have any difference? I see "value with type" being used more, and am wondering if `_returnValueWithType` would make more sense here? (but I have not fully digested the changes)